javascript/javascript1/week1/dogageCalculator/dogage.js

```diff
@@ -0,0 +1,13 @@
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2018;
+const dogYearFuture = 3000;
+const dogYear = (dogYearFuture - dogYearOfBirth);
+
+const shouldShowResultsInDogYears = true; 
+if (dogYear > 10) {
+    console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture)
+}
```

Comment:
Your code outputs the same number if dogYear > 10 and in the else. That's not the point of the exercise. `shouldShowResultsInDogYears` is not used. This exercise is not finished.